features/step_definitions/storage_class.rb

```diff
@@ -248,7 +248,8 @@
   when 'openstack'
     provisioner = 'cinder'
   else
-    raise "Unsupported platform `#{platform}`"
+    logger.error "Unsupported platform `#{platform}`"
```

Comment:
If this kind of behavior is expected, perhaps it should be `logger.warn`, a warn rather than a error. WDYT ?

Reply:
Fixed, thanks.!